CMakeLists.txt

```diff
@@ -417,3 +417,8 @@ if(BUILD_PACKAGES)
   include("Package.cmake")
   include(CPack)
 endif()
+
```

Comment:
This might not be neccessary?

Reply:
Do you mean `include(CTest)` is not neccessary?

Reply:
But current cmakelist already contains include(CTest)?

Reply:
It should be there: https://github.com/Cytnx-dev/Cytnx/blob/f43dcbd960b955611cddcbc036cc07e113dbbb8b/CMakeLists.txt#L144
I don't know why it doesn't show in the comparison tab.